components/timezone/src/lib.rs

```diff
@@ -90,6 +90,7 @@ extern crate alloc;
 mod error;
 mod ids;
 pub mod provider;
+pub mod scaffold;
```

Comment:
`#[doc(hidden)]`?

Reply:
But it is public, not just ICU4X-internal, and I really do not like `#[doc(hidden)]`. I'm happy to slap unstable warnings on the module and everything inside of it, similar to the provider modules.